src/DataTable.php

```diff
@@ -307,6 +319,8 @@ public function boot(): void
 
     public function render(): View|Factory|Application
     {
+        $this->forceRender = false;
```

Comment:
This means if one wants to set forceRender = true, the render() method needs to be overwritten.

Reply:
intended, forceRender is like skiprender, only valid per request…

if you have a function that should trigger a render you have to set forceRender in this function to true.